src/selection.js

```diff
@@ -149,13 +153,12 @@ export const findPositionOfNodeBefore = selection => {
     if (parent) {
       return parent.pos;
     }
-    return maybeSelection.$from.pos + 1;
+    return maybeSelection.$from.pos;
   }
 };
```

Comment:
how does this method differ to `selection.$from.before()`

Reply:
imagine you have a table before the cursor:
```
[doc]
  [p]text[/p]
  [table][/table]{gap-cursor}[p]text[/p]
[/doc]
```
This method gives the position of the table (the node before the cursor), whereas `selection.$from.before()` would throw an error on the depth=0. We use it in gap cursor to remove the node on Backspace